### ISSUE TITLE
Tab Order Not Respected After Filter/Sort Operations

### ISSUE THREAD

**Issue by chrisalexander55** [metafizzy/isotope#1108]
After Isotope's filter or sort method completes and a user tabs through the page, elements that were filtered/sorted are not included in the DOM tab order. Is this a bug? If not, what guidance can be given to make this work? I have a handler for the arrangeComplete event but really don't want to implement a JS-driven tabbing solution as this can be messy to debug/maintain. Thank you in advance.

**Test Case 1: Non-Assistive Technology User** 
1. After following the link below, tab through the page until one of the filter or sort buttons.
2. Tab through remaining page elements.
3. Note that the tab focus will never hit any of the element cards.

**Test Case 2: Assistive Technology User**
1. Using Chrome install and enable Chrome Extension ChromeVox. You can also try with JAWS on WIN.
2. After following the link below, tab through the page until one of the filter or sort buttons is announced.
3. Hit the enter key.
4. The elements will filter/sort.
5. Tab through remaining page elements.
6. Note that the tab focus will never hit any of the element cards. This type of user will never know of the element grid changes. This fails WCAG 2.0 compliance.

https://codepen.io/desandro/full/nFrte/

**Comment by desandro** [comment]
Thank you for reporting this issue. I'm sorry to say that no, Isotope does not support the accessibility features currently. **+1 this issue or add a reaction** if you would like to see these issues resolved in Isotope.

**Comment by julzmon** [comment]
+1

**Comment by alindt** [comment]
Could this be implemented by adding `tabindex="-1"` to the hidden elements after filtering?
